data/json/items/comestibles/carnivore.json

```diff
@@ -595,14 +595,33 @@
     "name": { "str": "cooked piece of lung", "str_pl": "cooked pieces of lung" },
     "description": " Prepared in this way, it's a chewy grayish lump of flavorless connective tissue.  It doesn't look any tastier than it did raw, but the parasites are all cooked out.",
     "parasites": 0,
-    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+    "extend": { "flags": [ "EATEN_HOT" ] }
+  },
+  {
+    "id": "mutant_lung",
+    "type": "COMESTIBLE",
+    "copy-from": "lung",
+    "name": { "str": "piece of raw mutant lung", "str_pl": "pieces of raw mutant lung" },
+    "snippet_category": "mutant_lung_desc",
+    "fun": -15,
+    "proportional": { "price_postapoc": 0.5 },
+    "extend": { "flags": [ "BAD_TASTE" ] },
+    "vitamins": [ [ "mutant_toxin", 10 ] ]
+  },
+  {
+    "id": "mutant_lung_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "lung_cooked",
+    "name": { "str": "cooked piece of mutant lung", "str_pl": "cooked pieces of mutant lung" },
+    "snippet_category": "cooked_mutant_lung_desc",
+    "fun": -10
```

Comment:
Apparently `lung_cooked` is alread -10 fun and you're using copy-from so you don't need to set it to -10 again